crud.rb

```diff
@@ -1,32 +1,65 @@
 require './create'
+require './read'
+require './delete'
+require './update'
 
-puts 'CRUD do João, Gerencie aqui seus posts'
+      puts
+      puts 'CRUD do João, Gerencie aqui seus posts'
+      puts
+      puts 'Digite o número referente a opção que deseja utilizar!'
+      puts
+      puts '1 - Criar Post'
+      puts '2 - Ler Post'
+      puts '3 - Alterar Post'
+      puts '4 - Deletar Post'
 
-puts 'Digite o número referente a opção que deseja utilizar!'
-puts
-puts '1 - Criar Post'
-puts '2 - Ler Post'
-puts '3 - Alterar Post'
-puts '4 - Deletar Post'
+      puts 'Informe a opção:'
+      action = gets.chomp
```

Comment:
now menu is called from crud (only with instructions) and variable action is on crud.rb